src/GraphQLController.php

```diff
@@ -45,7 +45,10 @@ public function query(Request $request, string $schema = null): JsonResponse
 
         // Complete each query in order
         foreach ($inputs as $input) {
-            $completedQueries[] = $this->executeQuery($schema, $input);
+            $completedQueries[] = $this->executeQuery(
+                $schema,
+                ($input['query'] ?? null) ? $input : ['query' => '']
+            );
```

Comment:
In your tests there's still always a `'query'`, so even if this fixes it, the tests don't proof it. Can you please extend them?

Reply:
This is indeed a good point! When examining this in more detail, I found that a fallback for a completely empty request (which was assumed to be a batch request) was probably also a good idea. Let me know if this is along the lines you have in mind.